moveit_ros/moveit_servo/src/utils/common.cpp

```diff
@@ -255,6 +255,22 @@ void updateSlidingWindow(KinematicState& next_joint_state, std::deque<KinematicS
   joint_cmd_rolling_window.push_back(next_joint_state);
 }
 
+/**
+ * @brief Composes a Float64MultiArray message for controllers requiring Float64MultiArray input.
+ * 
+ * This function converts the given joint state into a Float64MultiArray message.
+ * 
+ * @param servo_params Configuration parameters for the servo, including message type selection.
+ * @param joint_state The joint state to be converted into the message.
+ * @return A Float64MultiArray message containing the joint state data.
+ * 
+ * Example:
+ * @code
+ * auto message = composeMultiArrayMessage(servo_params, joint_state);
+ * controller.publish(message);
```

Comment:
It may not be obvious to most people that `controller` in this case is actually a ROS 2 publisher. Maybe call this variable `controller_pub` or something like that?

Also looking at other examples, it appears you may need to indent what is inside the `@code` / `@endcode`?

For example: https://github.com/moveit/moveit2/blob/524ca4f15bb449e1f8d25223d9725ca7868c32ee/moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.hpp#L607-L610

Which renders to: https://moveit.picknik.ai/main/api/html/classplanning__scene__monitor_1_1LockedPlanningSceneRO.html#details